src/main/java/net/pms/dlna/MapFile.java

```diff
@@ -99,11 +184,29 @@ private void manageFile(File f) {
 					addChild(new ZippedFile(f));
 				} else if (configuration.isArchiveBrowsing() && (lcFilename.endsWith(".rar") || lcFilename.endsWith(".cbr"))) {
 					addChild(new RarredFile(f));
-				} else if (configuration.isArchiveBrowsing() && (lcFilename.endsWith(".tar") || lcFilename.endsWith(".gzip") || lcFilename.endsWith(".gz") || lcFilename.endsWith(".7z"))) {
+				} else if (
+					configuration.isArchiveBrowsing() && (
+						lcFilename.endsWith(".tar") ||
+						lcFilename.endsWith(".gzip") ||
+						lcFilename.endsWith(".gz") ||
+						lcFilename.endsWith(".7z")
+					)
+				) {
 					addChild(new SevenZipFile(f));
-				} else if ((lcFilename.endsWith(".iso") || lcFilename.endsWith(".img")) || (f.isDirectory() && f.getName().toUpperCase().equals("VIDEO_TS"))) {
+				} else if (
+					lcFilename.endsWith(".iso") ||
+					lcFilename.endsWith(".img") ||
+					f.isDirectory() &&
```

Comment:
We have lost some brackets in this reformatting, are you sure this is treated as identical logic? I would probably prefer to leave them in there anyway for readability

Reply:
@SubJunk I can add them back, I simply removed them because there were a lot of redundant ones there, which I thought was less readable. The order in which they are solved as ```&&``` before ```||```.

This means that ```A && B || C ``` will be processed as ```(A && B) || C```. You only need parenthesis if you want it to be ```A && (B || C)```.

Reply:
@SubJunk This is the old format:
```java
					(
						lcFilename.endsWith(".iso") ||
						lcFilename.endsWith(".img")
					) || (
						f.isDirectory() &&
						f.getName().toUpperCase(Locale.ROOT).equals("VIDEO_TS")
					)
```
The first parenthesis really make no sense, ```(A || B) || C``` is the same as ```A || B || C```. The second parenthesis isn't needed because of [operator precedence](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/operators.html), but I'll leave that in for clarity:
```java
					lcFilename.endsWith(".iso") ||
					lcFilename.endsWith(".img") || (
						f.isDirectory() &&
						f.getName().toUpperCase(Locale.ROOT).equals("VIDEO_TS")
					)
```

Reply:
Cool